IDE/Renesas/e2studio/RA6M3G/common/src/freertos_tcp_port.c

```diff
@@ -0,0 +1,44 @@
+#include "bsp_api.h"

```

Comment:
This file needs our wolfSSL copyright added to the top.

Reply:
https://github.com/wolfSSL/wolfssl/blob/91a9117e1b7e7ca316299cef6dfc2243aa127651/IDE/Renesas/e2studio/RA6M3G/common/src/freertos_tcp_port.c#L1-L22

Thanks, updated.